specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.AppConfiguration/AppConfiguration.tsp

```diff
@@ -4,43 +4,43 @@ namespace Microsoft.EventGrid.SystemEvents;
 /** Schema of the Data property of an EventGridEvent for a Microsoft.AppConfiguration.KeyValueModified event. */
 model AppConfigurationKeyValueModifiedEventData {
   /** The key used to identify the key-value that was modified. */
-  key?: string;
+  key: string;
 
   /** The label, if any, used to identify the key-value that was modified. */
```

Comment:
label, if any?

Reply:
confirmed this returns Null in the case of KeyValueModifiedEventData -- nullable and required

Reply:
![image](https://github.com/user-attachments/assets/daf06ab8-060e-4170-ac9b-077ffed4dab5)